backend/Dockerfile

```diff
@@ -21,7 +24,10 @@ COPY package.json package-lock.json ./
 RUN npm install --production
 
 COPY --from=builder /app/dist ./dist
+COPY --from=builder /app/config ./config
```

Comment:
why are we adding config since we removed to used envsafe?

Reply:
looks like leftover code form merging the envsafe change into the branch, i shall remove 🙏